src/NServiceBus.Core/Pipeline/Incoming/IncomingLogicalMessageContext.cs

```diff
@@ -36,7 +36,7 @@ public void UpdateMessageInstance(object newInstance)
                 return;
             }
 
-            var factory = this.Builder.Build<LogicalMessageFactory>();
+            var factory = Builder.Build<LogicalMessageFactory>();
```

Comment:
Same here, some light boy scouting should be allowed as long as it doesn't hurt code reviewing